### ISSUE TITLE
[BUG] Update the Todo tutorial to avoid race condition for OnlineUsers in UI

### ISSUE THREAD

**Issue by avallete** [hasura/learn-graphql#435]
# TL;DR
On network with a medium latency, the Todo App tutorial "realtime Online Users" feature can look buggy because of race condition.

# Description of the issue

Into the [Todo-App backend tutorial](https://github.com/hasura/learn-graphql/blob/master/tutorials/backend/hasura/tutorial-site/content/data-transformations.md) the time condition interval on the `last_seen` value for a user to be present in the view `online_users` is set to 30s.

Then, in the [Todo-App frontend applications docs](https://github.com/hasura/learn-graphql/search?l=Markdown&q=30000), the interval value is also set to 30s.

However, due to network latencies and other parameters, this setup may trigger some "UI Blinking" of the "Online Users" because of race condition between the "postgres view table check" and the "user update call" into the frontend code, making the "online" user disappear for a second from the `online_users` view. Even if this race condition isn't necessarily consistent (fast enough network may never see it appear), if it happens to an user trying out the tutorial on a network with a little latency that will deteriorate the onboarding experience.

# Possibles fix

Either:

- increasing the time interval on the `online_users` for the `last_seen` value (ex: set it to 40s instead of 30s). [impacted files](https://github.com/hasura/learn-graphql/search?q=00%3A00%3A30)

- decreasing the call to `updateLastSeen` interval into the frontend applications code (20000 instead of 30000). [impacted files](https://github.com/search?p=1&q=30000+repo%3Ahasura%2Flearn-graphql+-filename%3Aprotractor.conf.js+-filename%3Amain.js+-filename%3ANetworkManager.swift+-filename%3A1-subscription.md&type=Code)

I would recommend the second approach who allows us to add a brief comment explaining why give some extra time to your interval. I can make a PR but would like to discuss the best option to choose before that.

**Comment by praveenweb** [comment]
Hi @avallete 

Thanks for opening this issue. To solve the race condition issue, I would recommend your 2nd suggestion of reducing the interval at which requests are made than changing the backend.

Feel free to open a PR :)

**Comment by avallete** [comment]
> Hi @avallete
> 
> Thanks for opening this issue. To solve the race condition issue, I would recommend your 2nd suggestion of reducing the interval at which requests are made than changing the backend.
> 
> Feel free to open a PR :)

Thank's for your input, PR opened and ready for review ;)
